src/schema/nmdc.yaml

```diff
@@ -2494,8 +2470,8 @@ slots:
   env_package:
     aliases:
       - environmental package
-    #      mappings:
-    #        - mixs:env_package
+    notes:
```

Comment:
Correct, and we're not collecting it in the submission protocol anymore but we still need to fill out this slot so we have a way to track what kind of sample we have ... biosample doesn't tell us if it's soil or plant or water or sediment, but the env_package should